merlin/loader/ops/embeddings/tf_embedding_op.py

```diff
@@ -0,0 +1,248 @@
+import numpy as np
+import tensorflow as tf
+
+from merlin.core.protocols import Transformable
+from merlin.dag import BaseOperator
+from merlin.dag.selector import ColumnSelector
+from merlin.schema import ColumnSchema, Schema, Tags
+
+
+class TFEmbeddingOperator(BaseOperator):
+    """Create an operator that will apply a tf embedding table to supplied indices.
```

Comment:
Most of these are repeated with small tweaks, would be nice to be able to converge so we dont have three operators for the same thing just using different inputs.